auto_tag/detectors.py

```
@@ -196,19 +190,18 @@ def __init__(self, *args: str, **kwargs: Any) -> None:
 
     def validate_detector_params(self) -> NoReturn:
         """Check if all the parameters given to the detector make sens."""
         super().validate_detector_params()
 
-        if self. _compiled_regex is None:
+        if self._compiled_regex is None:
             raise exception.DetectorValidationException(
-                ('Patter: {} is not valid regex.'
-                 'it must be specified and compliant').format(self._pattern))
+                f"Patter: {self._pattern} is not valid regex.it must be specified and compliant"
+            )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)